cumulusci/core/source_transforms/transforms.py

```diff
@@ -335,6 +337,39 @@ def process_file(filename: str, content: str) -> T.Tuple[str, str]:
         return process_text_in_zipfile(zf, process_file)
 
 
+class StripUnwantedComponentsOptions(BaseModel):
+    package_xml: str
+
+    def get_package_xml(self) -> str:
+        return self.package_xml
+
+
+class StripUnwantedComponentTransform(SourceTransform):
```

Comment:
This looks great. Did using it in `deploy` work for the use case you needed?

Reply:
Yes it will use same `RemoveSourceComponents` class in `deploy` task